src/parser.rs

```diff
@@ -734,6 +734,17 @@ impl<'a> Parser<'a> {
                             Ok(Expr::CompoundIdentifier(id_parts))
                         }
                     }
+                    // string introducer https://dev.mysql.com/doc/refman/8.0/en/charset-introducer.html
+                    Token::SingleQuotedString(_)
+                    | Token::DoubleQuotedString(_)
+                    | Token::HexStringLiteral(_)
+                        if w.value.starts_with('_') =>
```

Comment:
@alamb Moved detection of Introduced String here. So I check it form `Token::Word` this seems to be the most generic solution. What do you think?

Reply:
Unfortunately, it turned out to be the wrong solution. In MySQL, it's possible to run a query like this one:
```
select _col1 'str' from test_table; 
```
In this case, `'str'` is just a column alias...
So, I need to check if the string introduces is one of the following stings: https://dev.mysql.com/doc/refman/8.0/en/charset-charsets.html with `_` prefix.

The question is where to check it. I may bring back `Token::StringIntroducer` or check it here, only when we have string token in front of the introducer. Personally, I would stick with the second option.

Reply:
@alamb What do you think, does it fit to the sqlparser-rs, to have such a check for mysql string introducers?

Reply:
I am sorry -- I am behind on sqlparser-rs reviews. I hope to catch up later this week